chart/templates/deployment.yaml

```diff
@@ -35,6 +34,9 @@ spec:
               memory: "1.5Gi"
             requests:
               memory: "768Mi"
```

Comment:
I'm not sure if the resource limits & requests should be made configurable through Helm.. I guess the chart authors (compsy) can decide what values are sensible, but maybe these values have to scale according to expected workload of the application.

Reply:
In either case I don't think it hurts to create values.yaml entries with these values as default. That way endusers of the helmchart can override them as they see fit.

Reply:
Ok, I'll fix it.

Reply:
Fixed.